content/virtualization.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Virtualization on Linux
+description: Create a VM on Linux with GNOME Boxes or Virtual Machine Manager
+keywords:
+  - VM
+  - libvirt
+  - GNOME Boxes
+  - virt-manager
+  - Virtual Machine Manager
+  - VMM
+  - VirtualBox
+  - VMWare
+  - Virtual Machine
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+tableOfContents: true
+---
+
+![GNOME Boxes with VMs](/images/gnome-boxes/boxes-full.png)
+
+This is a quick tutorial on running virtual machines, or VMs, on Linux. Virtual machines allow you to run other operating systems from within your primary operating systems. This is a great way to run applications that require older software, run applications that require an OS other than Linux, or to try out other operating systems.
+
+This article will cover the GNOME Boxes application for running virtual machines. GNOME Boxes uses the open source [libvirt](https://libvirt.org/) hypervisor, which is the software that manages and runs the VMs.
+
+## Set Up
+
+First things first, we will need to a Linux operating system installed on your computer. We will be using Pop!_OS as the operating system running the virtualization software, called the Host operating system. Then, we will use Ubuntu as the "Guest" operating system, which runs inside the VM.
```

Comment:
Minor scope - I would assume most people that come across this article will already be running Linux. This section is still useful, but it may be helpful to clarify with something like:

"This article assumes you are already running a Linux distribution. Instructions in this guide use Pop!_OS as the "Host" operating system: The operating system that runs the virtualization software. See instructions below if you aren't running Linux, or would like to install Pop!_OS to make things simple."

Reply:
I added a link to the Install Pop!_OS in a VM article to the links section, and made the suggested change.